current_pose: current_pose
desired_pose: goal
cmd_vel/denmpc: denmpc/cmd_vel
mpc_plan: denmpc/local_plan

control_period: 0.1

max_speed: 1.0 # m/s
max_steering_angle: 2.7 # rad
max_acceleration: 10.0 # m / s^2

distance_offset: 0.1
theta_offset: 0.2

horizon_diskretization: 10
horizon_length: 0.5
